rfc/rfc-000-lightweightRFCprocess.md

```diff
@@ -0,0 +1,76 @@
+
+Much inspiration for this proposal has been drawn from [Apache Geode's RFC process](https://cwiki.apache.org/confluence/display/GEODE/Lightweight+RFC+Process), which in turn draws from Phil Calçado’s [Structured RFC Process](https://philcalcado.com/2018/11/19/a_structured_rfc_process.html).
+
+All RFCs are submitted via PRs to the [github.com/projectriff/riff](https://github.com/projectriff/riff) repo which will be merged when:
+- The `Reviewed By` date has elapsed
+- The RFC author has summarized the discussion in the PR thread.
+- concensus has been reached about the state ("Accepted/Dropped").
+
+### Collaboration
+Comments and feedback should be provided on the PR.
+
+Authors should address all comments by the deadline. This doesn't mean every comment and suggestion must be accepted and incorporated, but they should be carefully read and responded to.
+
+Every RFC is in one of the following phases:
+* **Active**: The RFC has been drafted and is open for comments. When making the PR, the doc should start out in this state.
+* **Accepted**: All the comments have been addressed and the proposed changes have been agreed upon. The implementation may start after this point.
+* **Dropped**: The changes proposed on this RFC were not agreed upon and no implementation will follow.
+* **Superseded**: The changes proposed on this RFC aren't in effect anymore, the document may be kept for historical purposes and there is a new RFC that’s more current.
+
+### Approval
+The proposal should be posted with a date by which the author would like to have the approval decision. How much time is given to comment depends on the size and complexity of the proposed changes, but ideally should be from 2 days to a week.
+
+### When to write an RFC?
+There is always the option of going straight to a pull request. However, for larger changes, it is advisable to reduce the risk of rejection of the pull request by first gathering input from the community.
+
+It’s encouraged to write an RFC for any major change. A major change might be:
+* Addition of new feature or subsystem
+* Changes that impact existing, or introduce new, public APIs.
+* Changes that will introduce user-facing configuration or concepts.
+* Changes that need to be coordinated across multiple repositories.
+
+### How to write an RFC?
+1. Copy the RFC template (in same folder as this document) and write your proposal! It's up to the author's discretion to decide which sections in the template make sense for their proposal. Cover the problem the proposal is solving, who it affects, how you’re proposing to solve it, and answers to frequently asked questions. Explicitly listing the goals will also make it easier to evaluate whether the proposal was successful.
+2. Add your RFC to the riff/rfc source directory, update to the next unique number. 
```

Comment:
I don't see any responses to my comments from yesterday. For example, I asked how the next RFC number would be determined when there are unmerged (open and closed) PRs consuming RFC numbers.

Reply:
I think any PR, once opened, reserves an RFC number. So even if RFC-42 becomes DROPPED, it is merged as DROPPED and number 42 is gone.

Reply:
Yes, I know, but the problem is how to work out which RFC numbers have been reserved. Presumably you have to scan all the PRs, opened and closed!?

Reply:
I deployed ZooKeeper to issue RFC numbers on demand. Kidding. I agree with Jurgen that we can keep this simple. If a collision is detected, the lower numbered PR (first to file) wins. If we skip a number, oh well.

RFC PRs shouldn't be closed without merging. They should be merged with a Dropped status.